Packs/CrowdStrikeFalcon/Playbooks/playbook-Command_and_Scripting_Interpreter.yml

```diff
@@ -0,0 +1,1717 @@
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "98":
+    id: "98"
+    taskid: c165d836-695b-4a31-8fe0-3d5fac6bef82
+    type: regular
+    task:
+      id: c165d836-695b-4a31-8fe0-3d5fac6bef82
+      version: -1
+      name: Set the Policy Actions
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.10/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: CrowdStrike.Detection
+      value:
+        complex:
+          root: incident
+          accessor: policyactions
+          transformers:
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+          - operator: Stringify
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '{"policyactions":'
+              suffix:
+                value:
+                  simple: '}'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1010,
+          "y": -890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "100":
+    id: "100"
+    taskid: de0ed1c3-2220-410b-8677-4eeae424d06d
+    type: regular
+    task:
+      id: de0ed1c3-2220-410b-8677-4eeae424d06d
+      version: -1
+      name: CommandLine Analysis
+      description: This script analyses a command line and provide a score based on the matched profiles.
+      scriptName: CommandLineAnalysis
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "101"
+    scriptarguments:
+      command_line:
+        simple: ${inputs.CommandLines}
+    reputationcalc: 2
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1410,
+          "y": -360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "101":
+    id: "101"
+    taskid: 8317cebf-6485-4687-8ef6-9f43ac3cb36b
+    type: regular
+    task:
+      id: 8317cebf-6485-4687-8ef6-9f43ac3cb36b
+      version: -1
+      name: Enrich Indicators Extracted From CommandLine
```

Comment:
either you check if you found indicators before (the task will fail in case you didnt found any) or you continue on error - I prefer the first option

Reply:
Adding condition in advance to that task. Make sense